src/api/source/mvt.js

```diff
@@ -37,15 +38,16 @@ export default class MVT extends Base {
      * @memberof carto.source
      * @IGNOREapi
      */
-    constructor(templateURL, metadata = new Metadata()) {
+    constructor(templateURL, metadata = new Metadata(), layerId = undefined) {
```

Comment:
layerId = undefined does nothing sine by default parameters are "undefined"

Reply:
I know, but I  put it to be explicit about the "optionalness" of the parameter

Reply:
ok, but in this case and since it`s a public method we should [update jsdoc too](http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values)

Reply:
:+1: